### PR TITLE
style: add issues template

### PR DESCRIPTION
**Pull request by YamenMerhi**
# What does this PR introduce?

## Style 🎨

- Add Issue templates

> Suggesting to have a Discussion Tab where we can answer questions related to the code without forwarding to LUKSO's discord sever

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        Please check the [issues tab](https://github.com/ERC725Alliance/ERC725/issues) to avoid duplicates.
+        Remember, this is not a place to ask for help debugging code. For that, we welcome you in the [LUKSO's discord server](https://discord.com/channels/359064931246538762/585786253992132609)!
+        Thanks for taking the time to fill out this bug report!
+    type: markdown
+  - attributes:
+      label: "Describe the issue:"
+    id: what-happened
+    type: textarea
+    validations:
+      required: true
```
Comment (skimaharvey) [inline .github/ISSUE_TEMPLATE/bug_report.yml:18]: may be a field for expected behavior? 
```yml
  - attributes:
      label: "Expected behavior:"
    id: expected-behavior
    type: textarea
    validations:
      required: true
  ```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - attributes:
```
Comment (skimaharvey) [inline .github/ISSUE_TEMPLATE/bug_report.yml:19]: also saw on trail of bits I think a part for: 
```yml
  - attributes:
      label: "Steps to reproduce the issue:"
    id: steps-to-reproduce
    type: textarea
    validations:
      required: true
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        Please check the [issues tab](https://github.com/ERC725Alliance/ERC725/issues) to avoid duplicates.
+        Thanks for taking the time to provide feedback!
+    type: markdown
+  - attributes:
+      label: "Describe the desired feature:"
+      description: Explain what the feature solves/improves.
+    id: feature-request
+    type: textarea
+    validations:
+      required: true
```
Comment (skimaharvey) [inline .github/ISSUE_TEMPLATE/feature_request.yml:18]: may be the motivation behind it?
```yml
  - attributes:
      label: "Rationale/Motivation:"
      description: "Explain why the feature is important and how it benefits the project."
    id: rationale
    type: textarea
    validations:
      required: true
 ```

**Review by skimaharvey — COMMENTED** [review]
looks good 🙏

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (CJ42) [reply .github/ISSUE_TEMPLATE/bug_report.yml:18]: agree on this field 👍

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      label: "Environment"
+      description: "Compiler version, Compiler options, Framework/IDE (e.g. Truffle/Hardhat/Remix), Operating System"
```
Comment (CJ42) [inline .github/ISSUE_TEMPLATE/bug_report.yml:21]: I would suggest adding npm + node.js version as well.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+body:
+  - attributes:
+      value: |
+        Please check the [issues tab](https://github.com/ERC725Alliance/ERC725/issues) to avoid duplicates.
+        Remember, this is not a place to ask for help debugging code. For that, we welcome you in the [LUKSO's discord server](https://discord.com/channels/359064931246538762/585786253992132609)!
+        Thanks for taking the time to fill out this bug report!
+    type: markdown
+  - attributes:
+      label: "Describe the issue:"
+    id: what-happened
```
[the remark this answers is not in the document]
Comment (skimaharvey) [reply .github/ISSUE_TEMPLATE/bug_report.yml:15]: could be that it happened only once but ideally can reproduce so may `what-happen` is better

File: .github/ISSUE_TEMPLATE/config.yml
[the remark this answers is not in the document]
Comment (YamenMerhi) [reply .github/ISSUE_TEMPLATE/config.yml:1]: To allow such links 
Screen Shot 2023-04-03 at 12 37 48 PM

**Review by skimaharvey — APPROVED** [review]

**Review by CJ42 — APPROVED** [review]

**Pull request merged**